### DISCUSSION TITLE
[Feature] ability to share pictures from memories easily

### DISCUSSION THREAD

**Discussion by tlowerison** [immich-app/immich#8741] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Ability to share pictures from / full slideshows of memories

The memory feature of n years ago today is so great, I don’t know if it exists on other default photo apps but it never popped up for me on iphotos. Would love to be able to share individual photos from those memories / share the full memory sequence of a particular year! I think it’d be an easy UX design, unsure of the actual engineering to do but generally a share button displayed while viewing a specific photo from a memory could share that specific photo, and in the multi year view where you can see the thumbnail for each year on this day (existing feature already, is where you initially click to check out these photos) perhaps each year‘s thumbnail could either have a small share button overlayed on it OR a triple dot button in case there were additional actions people would want to take on a specific year’s day (save to album, etc.)

Another easy alternative to the above mentioned options is to just include a button which links to the original photo in the main timeline. Then that main photo would support sharing.

But as of now, memory photos can only be shared via screenshot currently

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4496
